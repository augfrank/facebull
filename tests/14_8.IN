M0   C0   C1  2427
M1   C0   C2  2622
M2   C0   C3  2775
M3   C0   C4  2272
M4   C0   C6  2460
M5   C0   C7  2957
M6   C0   C8  2585
M7   C0   C9  2995
M8   C0   C10  2844
M9   C0   C11  2731
M10   C0   C13  2826
M11   C1   C0  2271
M12   C1   C3  2365
M13   C1   C4  2068
M14   C1   C5  2009
M15   C1   C6  2950
M16   C1   C7  2580
M17   C1   C8  2853
M18   C1   C10  2126
M19   C1   C12  2640
M20   C1   C13  2639
M21   C2   C0  2595
M22   C2   C1  2491
M23   C2   C3  2909
M24   C2   C5  2907
M25   C2   C6  2787
M26   C2   C8  2702
M27   C2   C9  2695
M28   C2   C10  2102
M29   C2   C11  2347
M30   C2   C12  2192
M31   C2   C13  2133
M32   C3   C0  2464
M33   C3   C1  2252
M34   C3   C2  2079
M35   C3   C4  2231
M36   C3   C5  2675
M37   C3   C6  2034
M38   C3   C7  2174
M39   C3   C8  2723
M40   C3   C10  2826
M41   C3   C11  2665
M42   C3   C12  2665
M43   C3   C13  2690
M44   C4   C0  2898
M45   C4   C1  2182
M46   C4   C2  2408
M47   C4   C3  2985
M48   C4   C5  2770
M49   C4   C6  2550
M50   C4   C7  2160
M51   C4   C8  2222
M52   C4   C11  2510
M53   C4   C12  2722
M54   C4   C13  2329
M55   C5   C0  2419
M56   C5   C2  2951
M57   C5   C3  2461
M58   C5   C6  2674
M59   C5   C7  2733
M60   C5   C8  2775
M61   C5   C9  2333
M62   C5   C10  2832
M63   C5   C11  2809
M64   C5   C12  2558
M65   C5   C13  2387
M66   C6   C0  2625
M67   C6   C1  2257
M68   C6   C3  2266
M69   C6   C4  2628
M70   C6   C5  2574
M71   C6   C7  2523
M72   C6   C8  2916
M73   C6   C9  2448
M74   C6   C10  2235
M75   C6   C11  2674
M76   C6   C12  2386
M77   C6   C13  2313
M78   C7   C0  2600
M79   C7   C1  2727
M80   C7   C2  2993
M81   C7   C3  2740
M82   C7   C4  2583
M83   C7   C5  2132
M84   C7   C6  2513
M85   C7   C8  2298
M86   C7   C9  2976
M87   C7   C10  2768
M88   C7   C11  2695
M89   C7   C12  2734
M90   C8   C0  2969
M91   C8   C1  2756
M92   C8   C2  2693
M93   C8   C3  2458
M94   C8   C4  2173
M95   C8   C5  2021
M96   C8   C7  2827
M97   C8   C9  2942
M98   C8   C10  2537
M99   C8   C11  2364
M100   C8   C12  2863
M101   C8   C13  2706
M102   C9   C0  2128
M103   C9   C1  2390
M104   C9   C2  2696
M105   C9   C3  2219
M106   C9   C4  2196
M107   C9   C5  2586
M108   C9   C6  2536
M109   C9   C7  2038
M110   C9   C8  2490
M111   C9   C11  2818
M112   C9   C12  2159
M113   C9   C13  2377
M114   C10   C0  2053
M115   C10   C1  2709
M116   C10   C2  2537
M117   C10   C3  2093
M118   C10   C4  2206
M119   C10   C5  2611
M120   C10   C6  2841
M121   C10   C7  2031
M122   C10   C8  2399
M123   C10   C9  2674
M124   C10   C12  2853
M125   C10   C13  2804
M126   C11   C0  2806
M127   C11   C1  2878
M128   C11   C2  2525
M129   C11   C3  2178
M130   C11   C4  2463
M131   C11   C5  2466
M132   C11   C6  2949
M133   C11   C7  2742
M134   C11   C9  2084
M135   C11   C10  2003
M136   C11   C12  2297
M137   C11   C13  2010
M138   C12   C0  2838
M139   C12   C1  2958
M140   C12   C2  2650
M141   C12   C3  2845
M142   C12   C4  2927
M143   C12   C5  2613
M144   C12   C6  2863
M145   C12   C7  2686
M146   C12   C8  2973
M147   C12   C9  2428
M148   C12   C10  2282
M149   C12   C11  2052
M150   C12   C13  2644
M151   C13   C0  2149
M152   C13   C1  2167
M153   C13   C2  2445
M154   C13   C4  2386
M155   C13   C5  2396
M156   C13   C6  2807
M157   C13   C7  2605
M158   C13   C8  2438
M159   C13   C9  2726
M160   C13   C10  2765
M161   C13   C11  2177
M162   C13   C12  2902
