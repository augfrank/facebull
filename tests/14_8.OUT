30252
4 13 31 34 50 61 68 85 95 112 114 135 149 152
